src/sass/utils/_variables.scss

```diff
@@ -220,7 +220,7 @@ $navbar-margin-bottom       : $line-height-computed;
 $navbar-border-radius       : 4px;
 $navbar-padding-horizontal  : floor($grid-gutter-width / 2);
 $navbar-padding-vertical    : (($navbar-height - $line-height-computed) / 2);
-$navbar-collapse-max-height : 340px;
+$navbar-collapse-max-height : none;
```

Comment:
Went through the same with the same issue on the front. not rly consider creating a PR for that hehe